rpa-manager/modules/ROOT/pages/processautomation-prepare-deployment-credential.adoc

```diff
@@ -0,0 +1,179 @@
+* If you have access to a Microsoft Azure account and want to create OAuth credentials for xref:rpa-builder::toolbox-mail-operations-mail-session-outlook-with-oauth.adoc[RPA Builder - Mail Session (Outlook with OAuth)], create an application for Outlook, https://learn.microsoft.com/en-us/exchange/client-developer/exchange-web-services/how-to-authenticate-an-ews-application-by-using-oauth[authenticate an EWS application by using OAuth^], and https://learn.microsoft.com/en-us/exchange/client-developer/legacy-protocols/how-to-authenticate-an-imap-pop-smtp-application-by-using-oauth[authenticate an IMAP, POP, or SMTP connection using OAuth^].
+* If you want to use RPA Builder to xref:rpa-builder::toolbox-variable-handling-credentials-for-oauth.adoc#generating-an-oauth-refresh-token[generate a refresh token for your OAuth credentials], ask an administrator to xref:usermanagement-manage.adoc#assign-users-rpa-builder-access[assign you RPA Builder access].
+
+== Create a Credential
+
+Create credentials with which a bot can log in to an external application during process runs.
+
+To create a credential:
+
+. Open the *Credential Pool* view of the *Process Automation* module.
+. Click *Create*.
+. [[form-create-credential]] Complete the *Create a New Credential* form:
+* [[form-create-credential-useraffiliation]] *User affiliation*
++
+Click to choose users that may view, edit, and use the credential. Add yourself so that you can also view, edit, and use the credential.
+* *User group affiliation*
++
+Click to choose user groups whose users may view, edit, and use the credential.
+* *Phase affiliation*
++
+Phases in which the credential can be deployed. When you create or change the credential, your privileges determine which phases the user can select:
++
+** *Test*
++
+The credential can be deployed in the test phase.
++
+** *Production*
++
+The credential can be deployed in the production phase.
+
+* *Type*
++
+Type of the global credential. The type of credential determines where you can use it. The original value defined in RPA Builder is typed. Select one of the following types:
++
+** *AWS Credentials*
++
+Use this credential for accessing Amazon Web Services. Fill in the client ID, client secret, and region. Using a session token is optional.
+** *OAuth Credentials*
++
+Use this credential for accessing the Microsoft Outlook email application via Microsoft Azure.
++
+*** *OAuth Host*
++
+The host address used to get the credentials. The default address for Outlook services is `https://login.microsoftonline.com/common/oauth2/v2.0/`. This address might change if the customer uses a self hosted service.
+*** *Client ID*
++
+To authenticate with the OAuth Host, an Azure application must be defined in the customer's space. This Azure application has an ID, which is unique in the entire OAuth Host space, and that allows users to explicitly grant or revoke access for this Azure application to their accounts.
```

Comment:
Is it possible to make this active? Define an Azure application....

Reply:
not sure, see above

Reply:
"define an Azure application in your space."